### PR TITLE
[feat] Create Sign up page

### PR DESCRIPTION
**Pull request by Hamdrive**
Created Sign up page for FueledUp™ using utility classes from PowertrainUI™
Consist of:
1. Username field
2. Password & Confirm password fields
3. Checkbox agreeing to T&C
4. Button to create account, another link to login

### CODE REVIEW

**Review by ZuberDunge — APPROVED** [review]
Looks good to me

**Review by abhij1607 — APPROVED** [review]
Good practices followed! Good work

File: pages/signup.html [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        <header class="max-width-1200 nav-main flex-row px-md mx-auto">
+          <div class="nav-logo">
+            <a href="../../index.html">FueledUp</a>
+          </div>
+          <div class="search-input">
+            <div class="fas fa-search search-icon"></div>
+            <label for="searchbar"></label>
+            <input
+              class="input-round input-md border-2"
+              type="text"
```
Comment (MaruthiPothuganti) [inline pages/signup.html]: Suggestion: input of **type = "search"** can be used instead of text for search bars as a best practice.

**Review by MaruthiPothuganti — APPROVED** [review]
Recommended changes: change of the value of type attribute of the search bar.
Everything else looks good to merge👍.

File: pages/signup.html (hunk 1 above)
Comment (Hamdrive) [reply pages/signup.html]: Thank you for pointing this out! Learnt something new today. Have made the suggested changes!

**Comment by Hamdrive** [comment]
Suggested changes made to branch, deploys running as expected. Merging to dev

**Pull request merged**